packages/codemods/transforms/v4/icon/index.js

```diff
@@ -0,0 +1,43 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+
+module.exports = function ({ source /*, path*/ }, { parse, visit }) {
+  const ast = parse(source);
+
+  const updateIsInlineBlockAttribute = (attributes) => {
+    const isInlineBlockAttr = attributes.find((a) => a.name === '@isInlineBlock');
+
+    if (isInlineBlockAttr) {
```

Comment:
quick consideration: 

_in theory_, since we're switching the logic, we need to assign a `@isInlineBlock` as `true` when there's not `@isInline` argument

_in practice_ this would cause a lot of noise in the consumers codebase, and I would advocate that we can simply ignore this and simply change the default (something to discuss in the RFC and gather data about, so we can justify our decision)

for context: I did a quick test in Cloud UI, and turned off this inline behaviour for the icons, and all the UI remained the same